packages/next/taskfile.js

```diff
@@ -510,6 +510,21 @@ export async function ncc_next__react_dev_overlay(task, opts) {
     })
     .target('dist/compiled/@next/react-dev-overlay/dist')
 
+  await task
+    .source(
+      relative(
+        __dirname,
+        require.resolve('@next/react-dev-overlay/dist/middleware-turbopack')
+      )
+    )
+    .ncc({
+      precompiled: false,
+      packageName: '@next/react-dev-overlay',
+      externals: overlayExternals,
+      target: 'es5',
+    })
+    .target('dist/compiled/@next/react-dev-overlay/dist')
```

Comment:
Is this overwritting the normal middleware?

Reply:
Nope, they get placed side-by-side.